src/features/patient_view/clinical_information/PDXTree.js

```diff
@@ -0,0 +1,116 @@
+import React from 'react'
+import SampleLabel from './SampleLabel'
+
+export default class PDXTree extends React.Component {
+  constructor(props) {
+    super(props)
+    this.renderNode = this.renderNode.bind(this)
```

Comment:
you probably don't need to bind to this.  arrow functions (which you are using below) don't change this "this" value.